blocks/library/latest-posts/index.php

```diff
@@ -92,7 +92,6 @@ function register_block_core_latest_posts() {
 			),
 			'align'           => array(
 				'type'    => 'string',
-				'default' => 'center',
```

Comment:
This one is tricky because we set all attributes using PHP code, but then override with JavaScript using hook. Should we add deprecation block for this case? /cc @youknowriad